lib/carrierwave/mongoid.rb

```diff
@@ -4,6 +4,7 @@
 require 'mongoid-grid_fs'
 require 'carrierwave'
 require 'carrierwave/validations/active_model'
+require 'byebug'
```

Comment:
I use and 'require' byebug locally to debug. I didn't want to pollute the code for you with it so I removed it before pushing the code.